### ISSUE TITLE
Tokens don’t work if they're UPPERCASE — esp. bad on Plain-Text Version

### ISSUE THREAD

**Issue by BillyHilly** [mautic/mautic#7545]
## Bug Description
Mautic Tokens are Case-Sensitive. Maybe they need to be that way. But I noticed if I have a token — let’s say `{contactfield=firstname}` — in an `<h1>` – `<h6>` tag, when I generate the Plain-Text version, the token comes in as All Caps, making it not work correctly in the plain-text version of the email.

| Q   | A
| --- | ---
| Mautic version | 14.0 
| PHP version | 7.1.29
| Browser | Safari 12.1

### Steps to reproduce
1. Create a segment new email, and add `{contactfield=firstname}` token into an `<h1>` – `<h6>` tag (doesn't matter which one, as long as it's an H tag)
2. Save and then go to Advanced —> Generate Plain-Text Version
3. Note how the Plain-Text version is correctly set to turn H tags into all caps, but it has no logic to detect tokens and it makes the token all caps as well.
4. Add yourself to the segment and send yourself the email. Inspect the plain text version and you’ll see the token text, not your name.

**Comment by RCheesley** [comment]
Hi there, as we are only providing fixes for Mautic 3.x going forward, can you confirm if this continues in Mautic 3.0.1?
Thanks!

**Comment by npracht** [comment]
Hi, without any feedback we cannot carry this issue open anymore. I'll close it and in case a new comment justify to re-open it, I'll be glad to do it :)

**Comment by BillyHilly** [comment]
Hello,

As previously mentioned, I'm using Mautic PRO now and assume I'm on the
latest version b/c of that. I can't see my version number anymore, so not
100% sure on that, but I think that's how it works.

I did a quick test, but I'm unable to see the "plain-text" version b/c my
mail client (Apple Mail) no longer has that as a View setting that made it
easy to check plain text.
Googled around and couldn't find a way to see the plain text version of my
test unfortunately.

So, in my test, I added a all caps token of {CONTACTFIELD=FIRSTNAME} to the
HTML version.
It worked correctly and showed my name.

I'm still not sure if  the Plain-Text version works correctly but I'm
assuming it does off this test.

If you do know of an easy way to view the plain-text version let me know
and I'll test again.
Or I can send you the test email to check out, just let me know what email
to send the test to.

Thanks!
Kevin
